controllers/ProposalController.php

```diff
@@ -647,26 +695,49 @@ private function checkIfCommentExists($commentId): Comment {
         return $comment;
     }
 
+    /**
+     * Check if User is owner of a comment.
+     * An admin can edit any comment.
+     *
+     * @param Comment $comment
+     */
     private function checkIfUserIsOwnerOfComment(Comment $comment) {
         $unauthorizedException = NotFoundHttpException::class;
 
+        if (MainController::getCurrentUser()->role == User::USER_ROLE_ADMIN) {
+            return;
+        }
+
         if(MainController::getCurrentUser()->id != $comment->author_id) {
             throw new $unauthorizedException();
         }
     }
 
-    private function checkIfCommentIsFromCurrentProposal(Comment $comment) {
+    /**
+     * Check if a comment is part of one proposal's comments.
+     * It checks if the user doesn't try to edit a comment coming
+     * from another proposal
+     *
+     * @param Comment $comment
+     */
+    private function checkIfCommentIsFromCurrentProposal(Comment $comment, int $proposalId) {
```

Comment:
Shouldn't it be : 
private function checkIfCommentIsFromCurrentProposal(Comment $comment, int $proposalId) 
{
instead of : 
private function checkIfCommentIsFromCurrentProposal(Comment $comment, int $proposalId) {

Reply:
Yep ! you're right